lib/validation.js

```diff
@@ -120,6 +127,32 @@ module.exports = function validation (yargs, usage, y18n) {
     }
   }
 
+  self.unknownCommands = function unknownCommands (argv, aliases, positionalMap) {
+    const commandKeys = yargs.getCommandInstance().getCommands()
+    const unknown = []
+    const currentContext = yargs.getContext()
+
+    if ((currentContext.commands.length > 0) || (commandKeys.length > 0)) {
+      argv._.slice(currentContext.commands.length).forEach((key) => {
+        if (commandKeys.indexOf(key) === -1) {
+          unknown.push(key)
+        }
+      })
+    }
+
+    if (unknown.length > 0) {
+      usage.fail(__n(
+        'Unknown argument: %s',
```

Comment:
Shouldn't this be `"Unknown command"`? The error as written looks too similar to the "unknown argument" output.

Reply:
you are 100% correct ... the reason I was thinking of using `Unknown argument:` was to avoid having to update all the languages yargs has been translated into, alternatively we could introduce the new phrase and let folks gradually submit translations.

Reply:
I think it's more important to be precise with error messages, even if it adds another untranslated string.

Reply:
👍 @mleguen mind making a PR for the french translation in a separate PR, as I know we've been fixing a few issues with translations recently.

Reply:
@bcoe No problem. I will do it when I will rebase the PR for french translations, once this one is merged.